README.md

```diff
@@ -88,13 +88,13 @@ Luckily that is straightforward:
 **On CUDA**
 
 ```bash
-pip install --index-url https://download.pytorch.org/whl/nightly/cu118 --pre 'torch>=2.1.0dev'
+pip install --index-url https://download.pytorch.org/whl/nightly/cu118 --pre 'torch==2.1.0.dev20230703'
```

Comment:
I don't remember how they handle it, but does this image persist? It might not be available in a few weeks, just saying :)